src/crimson/os/seastore/seastore_perf_counters.h

```diff
@@ -44,15 +41,6 @@ class PerfService {
     ss_plb.add_u64_counter(
       ss_tm_extents_allocated_bytes, "extents_allocated_bytes",
       "the total bytes of extents which have been allocated");
-    ss_plb.add_u64_counter(
-      ss_sm_bytes_written, "sm_bytes_written",
-      "the number of bytes written by segmentmanager");
-    ss_plb.add_u64_counter(
-      ss_sm_segments_released, "sm_segments_released",
-      "the number of segments released by segmentmanager");
-    ss_plb.add_u64_counter(
-      ss_sm_bytes_released, "sm_bytes_released",
-      "the number of bytes released by segmentmanager");
```

Comment:
all above are required by Sam, please make sure he agrees only use segments_released.

Reply:
in this change, all the implemented perf counters are preserved. we can always implement the missing ones on demand.

Reply:
Mmm, yeah.  Perhaps some form of crimson.seastore. prefixing scheme?  Something for later.